src/modules/renderPage.js

```diff
@@ -0,0 +1,10 @@
+const score = document.querySelector('.table');
+function loadSinglescore(player) {
+  const row = document.createElement('tr');
+  const item = document.createElement('th');
+  item.innerHTML = `${player.user}: ${player.score}`;
+  row.appendChild(item);
+  score.appendChild(row);
+}
```

Comment:
Kindly ensure to use arrow functions all through your project instead of regular functions as this is a requirement in this milestone👍.
<img width="499" alt="image" src="https://github.com/misalazeem/LeaderBoard/assets/64643995/2dcba751-72df-4203-9273-6c2046293ad9">